### DISCUSSION TITLE
Does Umami display the IP address of the requests?

### DISCUSSION THREAD

**Discussion by AhmedAburady** [umami-software/umami#3137] [Q&A]
I'm new to umami I'm using it in docker and everything is working as expected I was  wondering if it shows the IP address on the interface somewhere? I got the location of the request logged correctly but I can't seem to find the Client IP is there like ENV variable i should add or something?

**Comment by mikecao** [comment]
Umami does not store IPs.

**Reply by varac** [reply]
It's great that Umami doesn't *store*  IPs, but for debugging purposes, is there a way of showing the IP in the logs ?
